code/modules/nano/modules/ert_manager.dm

```diff
@@ -70,6 +70,7 @@
 		notify_ghosts("An ERT is being dispatched. Open positions: [slot_text]")
 		message_admins("[key_name_admin(usr)] dispatched a [ert_type] ERT. Slots: [slot_text]", 1)
 		log_admin("[key_name(usr)] dispatched a [ert_type] ERT. Slots: [slot_text]")
+		event_announcement.Announce("Attention, [station_name()]. We are attempting to activate an ERT. Standby.", "ERT Activation Attempt In Progress")
```

Comment:
The phrase "ERT Activation Attempt in Progress" sounds weird to me and it's just repeating the same information in the body of the announcement.  The header change I think makes it more clear at a glance what the announcement means; that CC has approved the ERT request and are attempting to dispatch one.  

I think it might sound better like this

### ERT Request Approved

Attention, NSS Cyberiad,

We are attempting to activate an ERT. Standby.

Reply:
well I originally did ERT Request Received 

but if admins decide to dispatch an ERT without a request for some reason it would say ERT _Request_ Approved/Received etc

Reply:
hmm maybe just "ERT Activation"?

Reply:
what about "ERT Protocol Activated" for Admin Dispatched ERTs?

Reply:
I actually like "ERT Protocol Activated" for both player requested ones and admin initiated ones.  The message itself then clarifies that it means they are attempting to activate one.

Reply:
Actually then the header would say ERT Protocol Activated, then "we are attempting to activate an ERT"

What about header: ERT Protocol Activated then the body "We are attempting to assemble an ERT"?